server/journeys/screen/start/controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response } from 'express'
+import { SanitisedError } from '../../../sanitisedError'
+import CsipApiService from '../../../services/csipApi/csipApiService'
+import PrisonerSearchService from '../../../services/prisonerSearch/prisonerSearchService'
+
+export class StartJourneyController {
+  constructor(
+    private readonly csipService: CsipApiService,
+    private readonly prisonerSearchService: PrisonerSearchService,
+  ) {}
+
+  redirectWithCsipData =
```

Comment:
This function will be used by pretty much every journey going forward. I'd move it to the shared /journeys/start/controller.ts file and remove this controller